android/capacitor.settings.gradle

```diff
@@ -1,51 +1,51 @@
 // DO NOT EDIT THIS FILE! IT IS GENERATED EACH TIME "capacitor update" IS RUN
 include ':capacitor-android'
-project(':capacitor-android').projectDir = new File('../node_modules/@capacitor/android/capacitor')
+project(':capacitor-android').projectDir = new File('../../capacitor/android/capacitor')
```

Comment:
Why does this point to the local repo if we are installing from npm?

Reply:
Oops, I was using local versions. This shouldn't be committed. I'll fix